entries/jQuery.getScript.xml

```diff
@@ -1,128 +1,136 @@
-  console.log( jqxhr.status ); // 200
-  console.log( "Load was performed." );
+	console.log( data ); // Data returned
+	console.log( textStatus ); // Success
+	console.log( jqxhr.status ); // 200
+	console.log( "Load was performed." );
 });
-    </code></pre>
-    <h4 id="handling-errors">Handling Errors</h4>
-    <p>As of jQuery 1.5, you may use <a href="/deferred.fail/"><code>.fail()</code></a> to account for errors:</p>
-    <pre><code>
+		</code></pre>
+		<h4 id="handling-errors">Handling Errors</h4>
+		<p>As of jQuery 1.5, you may use <a href="/deferred.fail/"><code>.fail()</code></a> to account for errors:</p>
+		<pre><code>
 $.getScript( "ajax/test.js" )
-  .done(function( script, textStatus ) {
-    console.log( textStatus );
-  })
-  .fail(function( jqxhr, settings, exception ) {
-    $( "div.log" ).text( "Triggered ajaxError handler." );
-});
-    </code></pre>
-    <p>Prior to jQuery 1.5, the global <code>.ajaxError()</code> callback event had to be used in order to handle <code>$.getScript()</code> errors:</p>
-    <pre><code>
-$( "div.log" ).ajaxError(function( e, jqxhr, settings, exception ) {
-  if ( settings.dataType == "script" ) {
-    $( this ).text( "Triggered ajaxError handler." );
-  }
-});
-    </code></pre>
-    <h4 id="caching-requests">Caching Responses</h4>
-    <p>By default, <code>$.getScript()</code> sets the cache setting to <code>false</code>. This appends a timestamped query parameter to the request URL to ensure that the browser downloads the script each time it is requested. You can override this feature by setting the cache property globally using <a href="/jquery.ajaxsetup/"><code>$.ajaxSetup()</code></a>: </p>
-    <pre><code>
-$.ajaxSetup({
-  cache: true
-});
-    </code></pre>
-    <p>Alternatively, you could define a new method that uses the more flexible <code>$.ajax()</code> method.</p>
-  </longdesc>
-  <example>
-    <desc>Define a $.cachedScript() method that allows fetching a cached script:</desc>
-    <code><![CDATA[
-jQuery.cachedScript = function( url, options ) {
+	.done( function( script, textStatus ) {
+		console.log( textStatus );
+	} )
+	.fail( function( jqxhr, settings, exception ) {
+		$( "div.log" ).text( "Triggered ajaxError handler." );
+	} );
+		</code></pre>
+		<p>Prior to jQuery 1.5, the global <code>.ajaxError()</code> callback event had to be used in order to handle <code>$.getScript()</code> errors:</p>
+		<pre><code>
+$( "div.log" )
+	.ajaxError( function( e, jqxhr, settings, exception ) {
+		if ( settings.dataType === "script" ) {
+			$( this ).text( "Triggered ajaxError handler." );
+		}
+	} );
+		</code></pre>
+		<h4 id="caching-requests">Caching Responses</h4>
+		<p>By default, <code>$.getScript()</code> sets the cache setting to <code>false</code>. This appends a timestamped query parameter to the request URL to ensure that the browser downloads the script each time it is requested. 
+			You can override this feature by using a settings object, as of jQuery 1.12:</p>
```

Comment:
Why the white space change here? It's all in one paragraph anyway so there will be no space in the rendered text.